subcommand/mesh-init/command_test.go

```diff
@@ -151,6 +191,51 @@ func TestRun(t *testing.T) {
 			envoyBootstrapContents, err := ioutil.ReadFile(envoyBootstrapFile.Name())
 			require.NoError(t, err)
 			require.NotEmpty(t, envoyBootstrapContents)
+
+			if c.checks != nil {
+				actualChecks, err := consulClient.Agent().Checks()
+				require.NoError(t, err)
+				for _, expCheck := range c.checks {
+					expectedAgentCheck := toAgentCheck(expCheck)
+					// Check for "critical" status. There is no listening application here, so checks will not pass.
+					expectedAgentCheck.Status = api.HealthCritical
+					// Pull the check type from the CheckID: "api-<type>" -> "<type>"
```

Comment:
nit: Might be worth double-checking other comments too.
```suggestion
					// Pull the check type from the CheckID: "api-<type>" -> "<type>".
```

Reply:
(I used Luke's suggestion, sorry!)